### ISSUE TITLE
create a vrt file in the folder when exporting an imageCollection to drive

### ISSUE THREAD

**Issue by 12rambau** [gee-community/geetools#316]
When an image collection is exported to asset, it fills an ImageCollection object which is by design made to deal with multiple images. 
I think it would be nice to have a vrt created along the TIFF images in the `toDrive` method as it would give the user a single entry point to the images using either `rasterio` or `rioxarray`.

It would most likely need an analysis function to build the said VRT on the fly before starting the export.
